packages/runtime/src/index.ts

```diff
@@ -1,7 +1,98 @@
-import type { GenerateResult, UnoGenerator, UserConfig, UserConfigDefaults } from '@unocss/core'
+import type { Arrayable, CSSObject, GenerateResult, UnoGenerator, UserConfig, UserConfigDefaults } from '@unocss/core'
 import { createGenerator, isString, toArray } from '@unocss/core'
 import { autoPrefixer, decodeHtml } from './utils'
 
+interface Theme {
+  width?: Record<string, string>
```

Comment:
These theme props are preset-specific, I am not very willing to hard-code them like this here. Could we figure out a way to reference the theme object from the preset somehow?

Reply:
I changed it to use Theme from preset-uno since it is listed in package.json dependencies anyway `"@unocss/preset-uno": "workspace:*"`, does this seem right?